Modules/xASL_module_ASL.m

```diff
@@ -81,6 +81,23 @@
     end
 end
 
+%##################### MULTI PLD & MULTI TE #############
```

Comment:
I think we can get rid of this part, then you don't have to worry about conflicts. In the current develop there is this part ...

```matlab
% Check if HadamardType is defined, if not set to 0
if ~isfield(x.modules.asl,'HadamardType') 
    x.modules.asl.HadamardType=0;
end
```

... because of #717 though. Maybe we can change that to `x.modules.asl.bHadamard`.